packages/daemon/src/daemon.js

```diff
@@ -186,46 +189,34 @@ const makeDaemonCore = async (
   const localNodeId = deriveId('node', rootEntropy, cryptoPowers.makeSha512());
   console.log('Node', localNodeId);
 
-  const knownPeersFormulaNumber = deriveId(
-    'peers',
-    rootEntropy,
-    cryptoPowers.makeSha512(),
-  );
-  const knownPeersId = formatId({
-    number: knownPeersFormulaNumber,
-    node: localNodeId,
-  });
-  await persistencePowers.writeFormula(knownPeersFormulaNumber, {
-    type: 'known-peers-store',
-  });
+  // We generate formulas for some entities that are presumed to exist
+  // because they are parts of the daemon's root object.
```

Comment:
```suggestion
  // because they are part of the daemon's root object.
```

Reply:
I think the cardinality of “formulas”, “entities”, “are”, and “parts” should agree here.